docs/patterns/card/index.md

```diff
@@ -160,5 +199,17 @@ A card can be used in light and dark themes.
     padding-inline: 0;
     padding-block: 0;
   }
+
+  rh-card.custom-light-theme {
+    --rh-color-border-subtle-on-light: #EF6461;
+    --rh-color-surface-lightest: #0081AF;
+    --rh-color-text-primary-on-light: #30292F;
+  }
+
+  rh-card.custom-dark-theme {
+    --rh-color-border-subtle-on-dark: #988B8E;
+    --rh-color-surface-darkest: #210B2C;
+    --rh-color-text-primary-on-dark: #F5FFC6;
```

Comment:
Looking at another existing custom color scheme (the Command Line Heroes site), this can use:
border: `#5e40be`
surface: `#261a4c`
text: `#e8e4f5`

Reply:
Those look a lot better.  Thank you!